src/patternfly/components/Timestamp/timestamp.scss

```diff
@@ -0,0 +1,24 @@
+.pf-c-timestamp {
+  --pf-c-timestamp--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-timestamp--Color: var(--pf-global--Color--200);
+  --pf-c-timestamp--TextAlign: center;
+  --pf-c-timestamp--Cursor: pointer;
+  --pf-c-timestamp--TextDecorationLine: underline;
+  --pf-c-timestamp--TextDecorationStyle: dashed;
+  --pf-c-timestamp--TextDecorationThickness: var(--pf-global--BorderWidth--sm);
+  --pf-c-timestamp--TextUnderlineOffset: #{pf-size-prem(4px)};
```

Comment:
> Because the timestamp would have a tooltip/popover to display the UTC (or some other custom, alternative datetime value), would it make sense to have this component behave more similarly to the [description list with term help text example](https://www.patternfly.org/v4/components/description-list#term-help-text) by making the timestamp a button?

Looks to me like the design guidance presents these styles when the timestamp has a tooltip/popover. I'd move these to a `.pf-m-help-text` modifier.